components/tidb_query_vec_expr/src/impl_string.rs

```diff
@@ -869,6 +897,74 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_lpad() {
+        let cases = vec![
```

Comment:
Can you add more corner cases, e.g. a too large `len`, a good `arg` & a good `len` but w/ an empty `pad`?

Reply:
Sure